packages/vant-area-data/package.json

```diff
@@ -8,7 +8,8 @@
   "exports": {
     ".": {
       "import": "./dist/index.esm.mjs",
-      "require": "./dist/index.cjs.js"
+      "require": "./dist/index.cjs.js",
+      "types": "./dist/index.d.ts"
```

Comment:
According to TypeScript document, the "types" condition should always come first in "exports".

See: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-7.html

![image](https://github.com/youzan/vant/assets/7237365/54310ff4-da90-467a-bcc8-d6360361a3b5)

Reply:
You mean I should put `types` first in `exports`, right?

Reply:
yes

Reply:
Or  do we need to add `types` like below
```json
"exports": {
        ".": {
            "import": {
                "types": "./dist/index.d.ts",
                "default": "./dist/index.esm.mjs"
            },
            "require": {
                "types": "./dist/index.d.ts",
                "default": "./dist/index.cjs.js"
            },
        }
    },
```